randovania/games/dread/gui/dialog/game_export_dialog.py

```diff
@@ -84,6 +81,35 @@ def romfs_validation(line: QtWidgets.QLineEdit):
 
 
 class DreadGameExportDialog(GameExportDialog, Ui_DreadGameExportDialog):
+    def get_path_to_ryujinx(self) -> Path:
+        ryujinx_path_tuple = ("Ryujinx", "mods", "contents", "010093801237c000")
+        current_system = platform.system()
+        if current_system == "Windows":
+            return Path(os.environ["APPDATA"], *ryujinx_path_tuple)
+        elif current_system == "Linux":
```

Comment:
Maybe this entire function could be done using `match`?

Reply:
Addressed